cpp/src/io/orc/orc.h

```diff
@@ -470,16 +470,22 @@ class ProtobufWriter {
  public:
   ProtobufWriter() { m_buf = nullptr; }
   ProtobufWriter(std::vector<uint8_t>* output) { m_buf = output; }
-  void putb(uint8_t v) { m_buf->push_back(v); }
+  void put_byte(uint8_t v) { m_buf->push_back(v); }
+  template <typename T>
+  void put_bytes(T const& values)
```

Comment:
`put_uint` returns the number of bytes added. Would it make sense to have that in `put_bytes` and `put_byte` as well? This could remove some of the magic addition to the size and keep things looking consistent.

Reply:
Good suggestion; done.